### PR TITLE
feat(client): search bar on all the pages

### PR DESCRIPTION
**Pull request by KarinKazarian**
#

## Changes this PR introduces

- search bar on all the pages

## How to test this PR

- 

## Notes for reviewers

-

## Checklist

- [x] I renamed the title of my pull request to follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [x] I have added tests to cover my changes (if applicable).

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-rh7cr5 Add search and filter on all other pages](https://app.clickup.com/t/rh7cr5)

**Comment by DavidLiang01** [comment]
I think you forgot to put a search bar in the bike tab of the quality management page

**Review by tiffzeng — DISMISSED** [review]
🎉🎉🎉

**Review by gordonpn — COMMENTED** [review]
Wow, search bar is a great feature!

In addition to David's comment. Could you also do 2 unrelated changes on this PR?

1. Change the page title of the scheduling page from "Production" to "Scheduling"

https://github.com/Mackbellemore/soen-390-team07/blob/65d1042865a8911464d67ed6cc52fe5e1e1465b4/client/src/components/views/SchedulingPage.jsx#L11-L15

2. Increase the inactivity timeout to 5 mins or whatever you think is appropriate

https://github.com/Mackbellemore/soen-390-team07/blob/65d1042865a8911464d67ed6cc52fe5e1e1465b4/client/src/components/IdleMonitor.jsx#L20

**Review by gordonpn — COMMENTED** [review]
@KarinKazarian Could you add the search bar to the production page? Now that it's merged

**Review by DavidLiang01 — DISMISSED** [review]

**Comment by DavidLiang01** [comment]
Amazing feature!

File: client/src/components/views/SchedulingPage.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const SchedulingPage = () => {
   return (
     <>
       <Head>
-        <title>ERP - Production</title>
+        <title>ERP - Scheduling</title>
       </Head>
       <Heading fontSize={{ base: '12px', sm: '26px' }} textAlign="center" mt={5}>
         Production
```
Comment (gordonpn) [inline client/src/components/views/SchedulingPage.jsx]:
```suggestion
        Scheduling
```

Comment (KarinKazarian) [reply client/src/components/views/SchedulingPage.jsx]: I thought I fixed this 🤯

**Review by gordonpn — APPROVED** [review]

**Review by Mackbellemore — APPROVED** [review]

**Pull request merged**